ext/node/02_require.js

```diff
@@ -819,8 +819,28 @@
   }
 
   function createRequire(filenameOrUrl) {
-    // FIXME: handle URLs and validation
-    const filename = core.ops.op_require_as_file_path(filenameOrUrl);
+    let fileUrlStr;
+    // TODO(kt3k): Support node.js Url object
```

Comment:
I think we can get this from `node.nativeModuleExports["url"].Url`?

Reply:
Tried (Node specific) Url object, and then it seemed throwing `ERR_INVALID_ARG_VALUE`

```js
import url from "url";
import { createRequire } from "module";
const u = url.parse(import.meta.url);
const require = createRequire(u);
```

This fails with:
```shellsession
$ node a.mjs
node:internal/errors:477
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_ARG_VALUE]: The argument 'filename' must be a file URL object, file URL string, or absolute path string. Received Url {
  protocol: 'file:',
  slashes: true,
  auth: null,
  host: '',
  port: null,
  hostname: '',
  hash: null,
  search: null...
    at new NodeError (node:internal/errors:387:5)
    at createRequire (node:internal/modules/cjs/loader:1239:11)
    at file:///Users/kt3k/a.mjs:4:17
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:541:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12) {
  code: 'ERR_INVALID_ARG_VALUE'
}
```

So I'm removing the TODO comment instead.